src/main/java/io/github/thebusybiscuit/cscorelib2/protection/ProtectionManager.java

```diff
@@ -81,6 +82,9 @@ public ProtectionManager(Server server) {
 		if (server.getPluginManager().isPluginEnabled("BlockLocker")) {
 			registerModule(new BlockLockerProtectionModule());
 		}
+		if(server.getPluginManager().isPluginEnabled("Lands")){
```

Comment:
Please add a space before and after the condition to fit in with the styling of our code.